gradle/releaser.gradle

```diff
@@ -55,25 +55,40 @@ task copyReadme(type: Copy, group: "releaser helpers", description: "copies the
 	into rootProject.buildDir
 }
 
+task copyGettingStarted(type: Copy, group: "releaser helpers", description: "copies the gettingStarted.adoc in preparation for search and replace") {
+	from(rootProject.rootDir) {
+		include "docs/asciidoc/gettingStarted.adoc"
+	}
+	into rootProject.buildDir
+}
+
 task bumpVersionsInReadme(type: Copy, group: "releaser helpers", description: "replaces versions in README") {
+	def bomVersion = rootProject.findProperty("bomVersion")
 	def oldVersion = rootProject.findProperty("oldVersion")
 	def currentVersion = rootProject.findProperty("currentVersion")
 	def nextVersion = rootProject.findProperty("nextVersion")
 	def oldSnapshot = currentVersion?.replace("RELEASE", "BUILD-SNAPSHOT")
 
 	onlyIf { oldVersion != null && currentVersion != null && nextVersion != null }
 	dependsOn copyReadme
+	dependsOn copyGettingStarted
 
 	doLast {
 		println "Will replace $oldVersion with $currentVersion and $oldSnapshot with $nextVersion"
+		if (bomVersion == null) println "No `bomVersion` set, will not change bom version in gettingStarted.adoc"
+		else println "Will change bomVersion in gettingStarted.adoc to $bomVersion"
 	}
 	from(rootProject.buildDir) {
 		include 'README.md'
+		include "docs/asciidoc/gettingStarted.adoc"
 	}
 	into rootProject.rootDir
-	filter { line -> line
-		.replace(oldVersion, currentVersion)
-		.replace(oldSnapshot, nextVersion)
+	filter { line -> (bomVersion == null) ?
+		line.replace(oldVersion, currentVersion)
+			.replace(oldSnapshot, nextVersion) :
+		line.replace(oldVersion, currentVersion)
+			.replace(oldSnapshot, nextVersion)
+			.replaceAll("\\:reactorReleaseTrain\\: .*", ":reactorReleaseTrain: " + bomVersion)
```

Comment:
I suggest passing the attribute directly via the asciidoc plugin, ie https://github.com/reactor/reactor-core/blob/4bdfbd3bef81b1a2992057ceaf0892d4d22d823f/gradle/asciidoc.gradle#L44

Reply:
let's put it in the `gradle.properties` then, and a manual step on top of the README bump task

Reply:
updated